api/src/main/java/javax/observer/collection/ListChange.java

```diff
@@ -1,18 +1,28 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
 package javax.observer.collection;
 
 import java.util.List;
 
 /**
- * TODO
- * @author Hendrik Ebbers
+ * 
+ * @param <T> element type of the collection
+ * 
+ * @author Michael Hoffer (info@michaelhoffer.de)
  */
-public interface ListChange<E> extends CollectionChange<E, ObservableList<E>> {
-
-    int getFrom();
-
-    int getTo();
+public interface ListChange<T> extends CollectionChange<T>{
 
-    List<E> getRemovedElements();
+    /**
+     * @return the indices of the changed elements
+     */
+    int[] indices();
```

Comment:
ok, makes sense. Maybe we can provide a default implementation that returns an `IntStream`. I would love to have the API as stream, lambda, fluent friendy as possible.